.github/workflows/forge-ci.yml

```
@@ -81,9 +81,10 @@ jobs:
             ./broadcast
           key: build-${{ github.event.pull_request.head.sha || github.event.after || github.sha }}
       - name: Test
-        run: forge test -vvv
+        # Comment out the `etherscan` section to avoid missing env var complaints
+        run: sed -i '/\[etherscan\]/,/^\[/ s/^/#/' foundry.toml && FOUNDRY_PROFILE=test forge test
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2209:warning:1:55: Use var=$(command) to assign output (or quote to assign string)

(shellcheck)